maquetado/scss/title.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+  align-items: center;
+  display: flex;
+  height: $text-decoration-height;
+  font-size: $text-big;
```

Comment:
Remember order by alphabetically. Check this [styles guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#properties-order)